<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarSearchDialog</class>
 <widget class="QDialog" name="CarSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Cars</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>261</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Car ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Make:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Performance:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Handeling:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Instrumentation:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtCarID"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtMake"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtModel"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtPerformance"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtHandeling"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtinstrumentation"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>241</width>
     <height>192</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Saftey Security:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Design:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Audio:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Comfort:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Maintenance:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Warrenty:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Packages:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtSaftey"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtDesign"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtAudio"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtComfort"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtMaintenance"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txtWarrenty"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="txtPackages"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
